conda/environments/cuml_dev_cuda10.1.yml

```diff
@@ -6,42 +6,25 @@ channels:
 - conda-forge
 dependencies:
 - cudatoolkit=10.1
-- clang=8.0.1
-- clang-tools=8.0.1
-- cmake=3.14.5
-- numba>=0.46
-- cupy>=7,<8.0.0a0
-- cudf=0.16*
-- rmm=0.16*
-- cython>=0.29,<0.30
-- pytest>=4.6
-- gtest=1.10.0
-- pytest-timeout
-- scikit-learn>=0.21
-- umap-learn>=0.3.9
-- scikit-learn>=0.21
-- dask>=2.12.0
-- distributed>=2.12.0
-- dask-cuda=0.16*
-- dask-cudf=0.16*
-- dask-ml
-- ucx-py=0.16*
-- ucx-proc=*=gpu
-- nccl>=2.5
-- libcumlprims=0.16.0a200812
-- statsmodels
-- treelite=0.92
-- doxygen
-- sphinx
-- sphinx_rtd_theme
-- numpydoc
-- nbsphinx
-- recommonmark
-- faiss-proc=*=cuda
-- libfaiss
-- ipython
+- rapids-build-env=0.16
+- rapids-notebook-env=0.16
+- rapids-doc-env=0.16
+- cudf=0.16
```

Comment:
Previously we had this specified as "cudf=0.16*" - more for my education, is there a meaningful difference between the two specs?

Reply:
that has changed a couple of times in the past, once due to a past behavior in conda, and then version number of our packages. Currently the recommendation seems to be 0.16.* (see https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-pkgs.html#preventing-packages-from-updating-pinning) which is perfectly compatible with our nightly and release versioning (i.e. `0.16.0a200825`), so changed it back to that.